### DISCUSSION TITLE
Is there a way to prevent resizing of a custom Node's sub component? (disregarding zooming fn)?

### DISCUSSION THREAD

**Discussion by cdolek** [xyflow/xyflow#3747] [Q&A]
Let's say we have a custom node, and this custom node has a notice purposed div in it that opens up and closes per state conditions. I don't want this component to be resized when I zoom in/out. Is this something that is supported already?

Thank you

**Comment by sylvaindore** [comment]
Not sure if that's supported, but the way I would approach this is to add the notice to the controls instead of a custom node

**Reply by cdolek** [reply]
Thank you for your suggestion. I'm particularly interested in keeping the notice as part of the custom node for a more integrated user experience. The notice is contextually linked to the specific node, and its relevance is directly tied to the state of that node.

I was thinking maybe there could be a class name that already filters certain nodes out...
